rocket_profile.py

```diff
@@ -0,0 +1,23 @@
+from typing import Dict, List
+
+from label import TYPE_CHECKING, Label, default_labels
+
+if TYPE_CHECKING:
+    from main import MainApp
+
+
+class RocketProfile:
+    def __init__(self, buttons: Dict[str, str], labels: List[Label]):
+        self.buttons = buttons
+        self.labels = labels
+
+    def update_labels(self, main_window: "MainApp") -> None:
+        for label in self.labels:
+            getattr(main_window, label.name + "Label").setText(label.update(main_window))
+
+
+tantalus = RocketProfile({"Arm": "arm", "Status": "status"}, default_labels)
+co_pilot = RocketProfile(
+    {"Arm": "arm", "Halo": "halo", "Data": "data", "Status": "status"},
+    default_labels + [Label("Test", lambda main_window: "Hey")],
+)
```

Comment:
These are just dummy profiles for the moment. I'd personally like to change these to whatever buttons and labels the rockets in question would actually use (so we don't have a label that just says "Hey" in master), but if we don't know what the actual rocket profiles will look like yet. this is fine for now.

Reply:
You could add a "test separation" button to tantalus and "tank pressure", "tank temp", "chamber pressure", "chamber temp" to co-pilot.

Reply:
Sounds promising. And what would their update methods look like?

Reply:
Sounds promising. And what would their update methods look like?

Reply:
Sounds promising. And what would their update methods look like?

Reply:
Dont really have that functionality yet. You could add a no-op that makes it clear it is a no-op/unimplemented.